README.rst

```diff
@@ -13,37 +13,42 @@ Micro-frontend for the single-page payment/checkout process. This application su
 Getting Started
 ---------------
 
-**Prerequisite**
+This MFE is bundled with `Devstack <https://github.com/edx/devstack>`_, see the `Getting Started <https://github.com/edx/devstack#getting-started>`_ section for setup instructions.
 
-`Devstack <https://edx.readthedocs.io/projects/edx-installing-configuring-and-running/en/latest/installation/index.html>`_.  If you start Devstack with ``make dev.up.ecommerce`` that should give you everything you need as a companion to this frontend.
+Once set up, this frontend and all its prerequisites can be started with:
 
-**Installation and Startup**
+.. code-block::
 
-1. Clone this repo:
+  cd devstack
+  make dev.up.frontend-app-payment
 
-  ``git clone https://github.com/edx/frontend-app-payment.git``
+The frontend runs at `http://localhost:1998 <http://localhost:1998>`_, though it is rare to interact with it directly.  Rather, log into the LMS at http://localhost:18000/login and enroll in a course, choosing "upgrade to verified" to be taken to the checkout page.
```

Comment:
nit: `Upgrade to Verified` capitalization

Reply:
Note: this button will say "Upgrade" when ticket REV-2129 goes out (included in our current sprint)

Reply:
I changed this to be a less specific directive for now, since I don't know when those changes will land.